src/selectors/entities/teams.ts

```diff
@@ -48,15 +49,15 @@ export const getCurrentTeam = reselect.createSelector(
     }
 );
 
-export function getTeam(state, id) {
+export function getTeam(state: GlobalState, id: string): Team {
     const teams = getTeams(state);
     return teams[id];
 }
 
 export const getCurrentTeamMembership = reselect.createSelector(
     getCurrentTeamId,
     getTeamMemberships,
-    (currentTeamId, teamMemberships) => {
+    (currentTeamId: string, teamMemberships: {[x: string]: TeamMembership}): TeamMembership => {
```

Comment:
0/5 on this, just FYI I added a type called Dictionary<T> which defines `{[x: string]: T}`